api/routes/api.php

```diff
@@ -54,9 +54,16 @@
         'prefix' => '/event-types',
     ], function () {
         Route::get('/', 'EventTypeController@index');
+        Route::get('/nickname/{nickname}', 'EventTypeController@getEventTypesByNickname');
         Route::get('/{id}', 'EventTypeController@getEventTypeById');
     });
 });
+Route::group([
+    'prefix' => '/event-types',
+    'namespace' => 'Api\\'
+], function () {
+    Route::get('/nickname/{nickname}', 'EventTypeController@getEventTypesByNickname');
+});
```

Comment:
Seems like the wrong group prefix.

Reply:
But if i move it to group with that same prefix and middleware, i would be unable to fetch event types.
So, as that backend for PUBLIC page, so there mustn't be any middleware.

Reply:
I do not mean moving to the existing group but moving to a separate route for the public page. But ok, it is up to you.